base/docs/helpdb.jl

```diff
@@ -1044,13 +1044,6 @@ Compute the minimum absolute values over the singleton dimensions of `r`, and wr
 minabs!
 
 doc"""
-    prevprod([k_1,k_2,...], n)
```

Comment:
These need to be manually removed from the rst docs. Again, keeping them in sync at the same time is a much clearer way to go right now.